.github/workflows/test-build-deploy.yml

```diff
@@ -319,12 +319,8 @@ jobs:
           name: Docker Images
       - name: Extract Docker Images from Archive
         run: tar xvf images.tar -C /
+      - name: Login to DockerHub
+        uses: grafana/shared-workflows/actions/dockerhub-login@13fb504e3bfe323c1188bf244970d94b2d336e86 # v1.0.1
```

Comment:
Note for reviewers, this is the latest release:
https://github.com/grafana/shared-workflows/commit/13fb504e3bfe323c1188bf244970d94b2d336e86